tests/test_code_format.py

```diff
@@ -5,26 +5,29 @@
 
 
 class TestCodeFormat(TestCase):
-    """ This test class should ensure all Harmony service Python code adheres
-        to standard Python code styling.
+    """This test class should ensure all Harmony service Python code adheres
+    to standard Python code styling.
 
-        Ignored errors and warning:
+    Ignored errors and warning:
 
-        * E501: Line length, which defaults to 80 characters. This is a
-                preferred feature of the code, but not always easily achieved.
-        * W503: Break before binary operator. Have to ignore one of W503 or
-                W504 to allow for breaking of some long lines. PEP8 suggests
-                breaking the line before a binary operatore is more "Pythonic".
+    * E501: Line length, which defaults to 80 characters. This is a
+            preferred feature of the code, but not always easily achieved.
+    * W503: Break before binary operator. Have to ignore one of W503 or
+            W504 to allow for breaking of some long lines. PEP8 suggests
+            breaking the line before a binary operatore is more "Pythonic".
+    * E203, E701: This repository uses black code formatting, which deviates
+                  from PEP8 for these errors.
```

Comment:
[See here for reference](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#pycodestyle)